src/sxastarter/README.md

```diff
@@ -1,4 +1,8 @@
-# Sitecore JSS Next.js Sample Application
+# Sitecore JSS Next.js Early Access Sample Application 
+
+## About
+This app includes some unreleased changes that are coming in 21.6.0.
+Check the [JSS changelog](https://github.com/Sitecore/jss/blob/dev/CHANGELOG.md) for the list of upcoming changes
```

Comment:
Should this point to release/beta (dev might have changes that won't be in 21.6.0)?

Reply:
makes sense, changed